app/src/main/java/com/example/reflector_android/Models/BlogRecyclerAdapter.kt

```diff
@@ -0,0 +1,35 @@
+package com.example.reflector_android.Models
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.example.reflector_android.R
+import com.example.reflector_android.network.Article
+import kotlinx.android.synthetic.main.article_item.view.*
+
+class BlogRecyclerAdapter(val articles: MutableList<Article>) : RecyclerView.Adapter<RecyclerView.ViewHolder>(){
```

Comment:
`class BlogRecyclerAdapter(val articles: MutableList<Article>) : RecyclerView.Adapter<RecyclerView.ViewHolder>() {`

https://developer.android.com/kotlin/style-guide

Add spacing between the end of your declaration `()` and the starting `{`